src/custom/CustomTooltip/customTooltip.tsx

```diff
@@ -25,39 +31,49 @@ function CustomTooltip({
   ...props
 }: CustomTooltipProps): JSX.Element {
   return (
-    <Tooltip
-      componentsProps={{
-        tooltip: {
-          sx: {
-            background: bgColor,
-            color: WHITE,
-            fontSize: fontSize || (variant === 'standard' ? '1rem' : '0.75rem'),
-            fontWeight: { fontWeight },
-            borderRadius: '0.5rem',
-            padding: variant === 'standard' ? '0.9rem' : '0.5rem 0.75rem',
-            boxShadow: 'rgba(0, 0, 0, 0.6) 0px 4px 10px, rgba(0, 0, 0, 0.5) 0px 2px 4px'
-          }
-        },
-        popper: {
-          sx: {
-            zIndex: 9999999999,
-            opacity: '1'
-          }
-        },
-        arrow: {
-          sx: {
-            color: bgColor
+    <SistentThemeProvider>
```

Comment:
you don't required here to wrap this provider.

Reply:
If we don't wrap this then the theme was not being applied properly and we can't access the correct tokens.